Elle624/FitLit-Refactor
# Feature/dom update

## Is this a fix or a feature?
Fix

## What is the change?
N/A

## What does it fix? 
Creates functionality in domUpdate file and. cleans up scripts

## Where should the reviewer start?
domUpdate.js: line 1
scripts.js: line 77

## How should this be tested?
Run `npm start` in terminal and open the local host and see that the page runs correctly 

### Screenshot (optional)


@atownse
@elle624
@lo-la-do-li
